modules/20-arithmetics/70-nan/description.en.yml

```diff
@@ -1,7 +1,26 @@
 ---
 
 name: NaN
-theory: 
-instructions: 
+
+theory: |
+  Some operations with infinity, for example, dividing infinity by infinity, return a weird result. In Math, this operation has no numerical equivalent. In JavaScript, NaN will return.
+
+  ```javascript
+  Infinity / Infinity; // NaN
+  ```
+
+  `NaN` is a special value "not a number" that usually indicates that a meaningless operation has been executed. Any operation with `NaN` returns `NaN`.
+
+  ```javascript
+  NaN + 1; // NaN
+  ```
+
+  `NaN` is a peculiar value. Although it means "not a number", in terms of types it is a number. Paradox. `NaN` is never a desired value, once it appears only as a result of errors. If you have got him, you need to track the moment at which the operation, which is not valid for numbers, was performed, and correct this piece of code.
```

Comment:
If you have got him, you need to track the moment at which the operation
=>
If you have encountered it, trace the moment where the operation